### PR TITLE
Add command to convert lead to client

### LINKED ISSUE #87: Convert leads to client
There is a need to convert leads to clients

Currently leads remains clients

### PR DESCRIPTION
**Pull request by LicongHuang**
Previously, there is no command to convert Lead to Client, and vice versa.

There is a need to convert lead to client.

Now, there is a command to convert lead to client

Closes #87

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -93,6 +95,12 @@ public Command parseCommand(String userInput) throws ParseException {
         case ViewCommand.COMMAND_WORD:
             return new ViewCommandParser().parse(arguments);
 
+        case ConvertClientToLeadCommand.COMMAND_WORD:
+            return new ConvertClientToLeadCommandParser().parse(arguments);
+
+        case ConvertLeadToClientCommand.COMMAND_WORD:
+            return new ConvertLeadToClientCommandParser().parse(arguments);
```
Comment (yuxunn) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:102]: Perhaps you could do testing on these cases

File: src/main/java/seedu/address/logic/parser/ConvertClientToLeadCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.ConvertClientToLeadCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new ConvertClientToLeadCommand object
+ */
+public class ConvertClientToLeadCommandParser implements Parser<ConvertClientToLeadCommand> {
```
Comment (yuxunn) [inline src/main/java/seedu/address/logic/parser/ConvertClientToLeadCommandParser.java:12]: Could add testing for `ConvertLeadToClientParser` and `ConvertClientToLeadParser` as well :)

**Review by yuxunn — CHANGES_REQUESTED** [review]
Just need some minor changes in test coverage, otherwise the feature looks good! I've tested it locally and it works fine! 👍

File: src/main/java/seedu/address/logic/parser/ConvertClientToLeadCommandParser.java (hunk 2 above)
Comment (LicongHuang) [reply src/main/java/seedu/address/logic/parser/ConvertClientToLeadCommandParser.java:12]: I have added test for ConvertLeadToClientParser and ConvertClientToLeadParser

**Review by yuxunn — APPROVED** [review]
Looks good 👍

**Pull request merged**